core/src/main/java/hivemall/evaluation/HitRateUDAF.java

```diff
@@ -71,9 +71,6 @@
                 + " - Returns HitRate")
 public final class HitRateUDAF extends AbstractGenericUDAFResolver {
 
-    // prevent instantiation
-    private HitRateUDAF() {}
-
```

Comment:
This prevents Spark from loading UDAFs by using reflection. Can we remove this?

Reply:
sure.